cmd/apps/registry_app.go

```diff
@@ -92,13 +92,13 @@ func MakeInstallRegistry() *cobra.Command {
 
 		htPasswd := fmt.Sprintf("%s:%s\n", username, string(val))
 
-		err = helm.AddHelmRepo("stable", "https://charts.helm.sh/stable", updateRepo)
+		err = helm.AddHelmRepo("stable", "https://helm.twun.io", updateRepo)
```

Comment:
The repo name here should be changed as well, from `"stable"` to `"twuni"`, for consistency with the chart path on L101.

Also FYI, the URL here could be adjusted to `"https://twuni.github.io/docker-registry.helm"` if you prefer. Both will work equally, just wanted to present the option.